README.md

```diff
@@ -1,22 +1,24 @@
-# Opencrypto Schema v0.9-beta
+# Opencrypto Schema v0.9
 
 [![Build Status](https://travis-ci.org/opencrypto-io/schema.svg?branch=master)](https://travis-ci.org/opencrypto-io/schema)
 
-Schemas for structured cryptocurrency-related data in [JSON Schema](http://json-schema.org/) standart.
+Schemas for structured cryptocurrency-related data in [JSON Schema](http://json-schema.org/) standard.
 
-**WARNING: Schema is currently unreleased and in heavy development. If you have some suggestion, feel free to contact us via [Issues](https://github.com/opencrypto-io/schema/issues).**
+We created this schema, because we need normalized way how to create, transfer and store data about crypto-assets and related ecosystem.
```

Comment:
We created this schema, because we need **to** normalized **the** way how to create, transfer and store data about crypto-assets and related ecosystem**s**.

Still sounds strange to me, I will talk to my friend if we can improve the style how it's written.

Reply:
We created this schema to unify the way for creation, transfer and storage of crypto-assets data and related ecosystems.

Reply:
👍